### PR TITLE
P0: a fresh project created every task into a column its workflow no longer declares

### PR DESCRIPTION
**Pull request by gsxdsm**
Highest-severity finding of the post-merge audit, and it is the **out-of-the-box** shape rather than an edge case.

## The defect

`createTask` resolves the intake column only as a by-product of materializing the project's default workflow. A project that has never **explicitly** set a default workflow has no persisted default row — so that materialization returns nothing, `resolvedEntryColumn` stays `undefined`, and the row falls through to the hard-coded `|| "triage"`.

Post-merge, that column does not exist in the default workflow. Measured, three creates on one store:

| create | column |
|---|---|
| no default row persisted | **`triage`** ← broken |
| default explicitly `builtin:coding` | `todo` |
| explicit `workflowId` | `todo` |

`builtin:coding` is the **implicit** default via `DEFAULT_WORKFLOW_ID`, and nothing writes a default-workflow row until an operator picks one. So this was **every new task on a fresh project.**

## What it costs

Triage discovery resolves intake **by trait**, so `isAtIntakeColumn` is false for a card sitting in `triage` while its workflow says `todo` — **the card is never admitted for planning.** It isn't in the hold column either, so hold-release ignores it. Only `reconcileUndeclaredTaskColumns` eventually re-homes it.

A newly created task is invisible to planning until that sweep runs. Not a permanent stall, but the first thing an operator does on a new project is create a task.

## The fix — three parts, and missing any one leaves it half-fixed

1. `resolveDefaultWorkflowIntakeColumn` falls back to `DEFAULT_WORKFLOW_ID` when no default row is persisted — the implicit default every other resolver already assumes.
2. Both create paths consult it as a **last** resort before the literal, so any path that already has an explicit column or a resolved entry column is untouched.
3. **`isIntakeColumn` honours the same fallback.** Without this the card lands in the right column but is classified *not*-intake and receives `generateSpecifiedPrompt` instead of the bootstrap seed — and triage admits a card only when its `PROMPT.md` reads as a seed, so it would sit in Planning already looking "planned". FN-8587's failure mode by another route.

`workflowId: null` ("No workflow") is excluded and asserted — there is no workflow whose intake could be resolved, so that path keeps the literal.

## Fixture drift, fixed with intent preserved

Seven tests asserted a created card lands in `triage`. None had their assertion merely retargeted:

- **`move-task-if-planning`, `delete-task-if-planning`** — the mechanism under test is the **live predicate**, not the column. Predicates and the "advanced" column now name where the card actually rests.
- **`task-lifecycle-e2e`, `activity-log-parity`, `mission-store`** — first-column and first-transition expectations.
- **`workflow-reconciliation-production-shape`** — the subtle one. Its filler must occupy the **target** workflow's capped `triage` entry column, but was created *before* the switch and so landed in the **project default's** intake. It now names its column explicitly, which makes the fixture independent of the project default — exactly the coupling that let it drift.
- **`store-create-intake-column`** — the "lands in triage" guard now names the invariant (the default workflow's *own* intake column) and keeps a `not.toBe("triage")` so a regression back to the literal still fails.

## Measured

Core package, against the 47-failure post-merge main baseline: **47 failed / 4413 passed — zero new failures.**

Three engine triage tests are red and are **not from this change**: verified by stashing these edits and re-running against clean main, where they fail identically. They arrived with #2515 and belong to the triage-fixture owner.

Gate 414 + 10 + 71 green. Lint clean.

🤖 Generated with [Claude Code](https://claude.com/claude-code)

## Summary by CodeRabbit

* **Bug Fixes**
  * New tasks now consistently start in the default workflow’s `todo` intake column, including fresh projects without persisted workflow settings.
  * Bootstrap `PROMPT.md` content is now created consistently for all supported task-creation paths.
  * Task movement and deletion behavior now correctly respects current columns and avoids acting on stale task data.
  * Workflow reconciliation and activity tracking now reflect the updated default task lifecycle.

* **Tests**
  * Expanded coverage for intake-column resolution, task lifecycle transitions, stale candidates, and workflow edge cases.

### CODE REVIEW

File: packages/core/src/task-store/task-creation.ts [hunk 1]
```diff
@@ -787,7 +847,7 @@ export async function _createTaskInternalImpl(store: TaskStore, input: TaskCreat
       // FNXC:CodingIdeasWorkflow 2026-07-05-19:45: land the task in its
       // workflow's manual intake column (e.g. Coding (Ideas) → "ideas") when
       // no explicit column is given (main FN-7591 parity).
-      column: input.column || options?.resolvedEntryColumn || "triage",
+      column: input.column || options?.resolvedEntryColumn || fallbackIntakeColumn || "triage",
```
[the remark this answers is not in the document]
Comment (gsxdsm) [reply packages/core/src/task-store/task-creation.ts:850]: Real, and fixed in 9a8779e12. Two copies of one predicate that disagreed, and the missing clause is exactly the one the backend path needed.

**Honest scope, because it matters more than the fix: I could not construct a failing test through a public API, and I am not claiming coverage for the divergence.**

The only in-tree caller of `_createTaskInternal` is `createTaskWithReservedIdImpl`, which passes its own `resolvedEntryColumn` through options — so the second disjunct already matches and this path's own `fallbackIntakeColumn` never decides. The reserved-id test I added passes both before and after the fix, and the comment in it says so rather than implying it caught something.

Fixed anyway, because the divergence is a latent bug rather than a style issue: a direct `_createTaskInternal` call — which the exported signature invites — lands the card in the resolved intake column and then classifies it NOT-intake, writing `generateSpecifiedPrompt` instead of the bootstrap seed. Triage admits a card for planning only when its `PROMPT.md` reads as a seed, so that card would rest in Planning already looking "planned" and never be planned. That is the same failure mode this PR exists to fix, reached one function over.

The new test still earns its place: it pins the reserved-id path's bootstrap contract, which nothing covered, so a future change that *does* route the fallback through this predicate cannot regress it silently.
